deploy/olm-catalog/3scale-community-operator/3scale-community-operator.v0.3.0.clusterserviceversion.yaml

```diff
@@ -107,29 +107,29 @@ spec:
     Documentation can be found on our [website](https://github.com/3scale/3scale-operator/blob/v0.3.0/doc/user-guide.md).
 
     ### Getting help
-    If you encounter any issues while using 3scale operator, you can create an issue on our [website](https://github.com/3scale/3scale-operator) for bugs, enhancements, or other requests.
+    If you encounter any issues while using 3scale community operator, you can create an issue on our [website](https://github.com/3scale/3scale-operator) for bugs, enhancements, or other requests.
 
     ### Contributing
     You can contribute by:
 
-    * Raising any issues you find using 3scale Operator
+    * Raising any issues you find using 3scale community Operator
     * Fixing issues by opening [Pull Requests](https://github.com/3scale/3scale-operator/pulls)
     * Improving [documentation](https://github.com/3scale/3scale-operator/blob/v0.3.0/doc/user-guide.md)
-    * Talking about 3scale Operator
+    * Talking about 3scale community Operator
 
     All bugs, tasks or enhancements are tracked as [GitHub issues](https://github.com/3scale/3scale-operator/issues).
 
     ### License
-    3scale Operator is licensed under the [Apache 2.0 license](https://github.com/3scale/3scale-operator/blob/master/LICENSE)
+    3scale community Operator is licensed under the [Apache 2.0 license](https://github.com/3scale/3scale-operator/blob/master/LICENSE)
 
   keywords:
     - 3scale
     - API
-  displayName: 3scale Operator
+  displayName: 3scale Community Operator
```

Comment:
If this name is what is displayed in the operator catalogue (e.g. operatorHub.io, or whatever) I think it would make sense to entirely omit "Operator" here. 

It's clear to me that I'm looking at a list of operators anyway, so I don't think the "Operator" in the name is much more than noise. 

(Just like in docker hub you wouldn't see "postgres docker image" but just "postgres" when looking at the list of container images)

Reply:
Agree, and Community will be removed as well because it is explicit from provider type. RH's Operatorhub displays `community` watermark when provider type is community